.circleci/smoke.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMPDIR="$(mktemp -d 2>/dev/null || mktemp -d -t 'apiejs-smoketest')"
+API_ELEMENTS="$(pwd)"
```

Comment:
I'd rather call this something like `PROJECT_DIR` so it's clearer what's in that variable, but YMMV

Reply:
Changed in 9889e3f